.github/workflows/release.yaml

```diff
@@ -4,8 +4,7 @@ on:
   push:
     tags: [ 'v*.*.*' ]
     
-permissions:
-  contents: write
+permissions: {}
```

Comment:
Is it explicitly set to empty? If not needed, we should remove it.

Reply:
`permissions: {}` sets `GITHUB_TOKEN` zero permissions. The reason we are using is ` goreleaser/goreleaser-action` is requires the token, if we not send that token, it throws error.

Reply:
updated the code.